clevertap-core/src/main/java/com/clevertap/android/sdk/network/NetworkManager.java

```diff
@@ -755,14 +782,21 @@ public boolean sendQueue(final Context context, final EventGroup eventGroup, fin
 
                 // Process the response body
                 if (eventGroup == EventGroup.VARIABLES) {
-                    CleverTapResponse cleverTapResponse = new CleverTapResponseHelper();
-                    cleverTapResponse = new ARPResponse(cleverTapResponse, config, this, validator,
-                            controllerManager);
-                    cleverTapResponse = new BaseResponse(context, config, deviceInfo, this, localDataStore,
-                            cleverTapResponse);
-                    cleverTapResponse.processResponse(null, body, this.context);
+                    processVariablesResponse(body);
                 } else {
-                    getCleverTapResponse().processResponse(null, body, this.context);
+                    // check if there is app launched/wzrk_fetch event
+                    boolean found = false;
```

Comment:
avoiding utils class and keeping functions in right place so that we avoid make a too large Utils class which can cause problems for modularity. If we need this function to be reused in other places i will then move it to common file.